lib/comgr/src/comgr-compiler.cpp

```diff
@@ -675,7 +688,7 @@ amd_comgr_status_t AMDGPUCompiler::ProcessFile(const char *InputFilePath,
   Argv.push_back("-o");
   Argv.push_back(OutputFilePath);
 
-  InProcessDriver TheDriver(LogS);
+  InProcessDriver TheDriver(LogS, this);
```

Comment:
If we are going to be out-of-process, I would rather not hide it under `InProcessDriver`. Can we just check for the language here in `ProcessFile` and invoke e.g. a separate static method implementing the spawning of `hipcc`?

Reply:
will do